packages/edit-navigation/src/components/menu-locations-editor/menu-select-control.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { SelectControl } from '@wordpress/components';
+
+export default function MenuSelectControl( {
+	location,
+	availableMenuIds,
+	onSelectMenu,
+} ) {
+	const [ selectedMenuId, setSelectedMenuId ] = useState( undefined );
```

Comment:
Any reason we can't default this to `location.menu`?

```js
const [ selectedMenuId, setSelectedMenuId ] = useState( location.menu );
```

Reply:
I did just that. I think the useEffect stuck from some code moving ... It's gone.